cli/pkg/kctrl/cmd/app/release/release_cmd_runner_test.go

```diff
@@ -0,0 +1,16 @@
+package release
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestReleaseCmdRunnerForKappCmd(t *testing.T) {
+	ReleaseCmdRunner := NewReleaseCmdRunner(os.Stdout, false, "", false, nil)
+	cmd := exec.Command("kapp", "deploy", "-f", "-", "-a", "pkg-test", "-y")
+	err := ReleaseCmdRunner.RunWithCancel(cmd, nil)
+	require.NoError(t, err)
```

Comment:
Can we check if Stdout is empty?

Reply:
Or rather pass a buffer instead so that we can be sure that the command was not run

Reply:
@100mik Added the check in this commit: https://github.com/carvel-dev/kapp-controller/pull/1638/commits/8185bfa41f9b7d373fac4511456800ae46ecc9c2